### PR TITLE
Created class Label with properties visibile on diagram and method add_item

### LINKED ISSUE #4: [3pt] Create a Label class - Team Member #1
### **Create a Label class with an association to the Item class (in a separate .rb file).**

- All Label class properties visible in [the diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png) should be defined and set up in the constructor method.
- **Implement methods:**

  - add_item method in the Label class
  - should take an instance of the Item class as an input
  - should add the input item to the collection of items
  - should add self as a property of the item object (by using the correct setter from the item object)

### PR DESCRIPTION
**Pull request by DeVuDeveloper**
Hi,
- added class Label
- visible properties on the diagram implemented in the constructor
- added public method add_item
- added label as a property of the item object
Update:
-changed attr writer to reader for id

### CODE REVIEW

File: src/classes/label.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require_relative 'item'
+
+class Label
+  attr_accessor :title, :color, :items
+  attr_writer :id
```
Comment (brytebee) [inline src/classes/label.rb]: - [ ] **Required:** Kindly change the id to be read-only, currently it can be edited after the label had been created.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @VuDej 👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

**Review by Beardless-sheik — APPROVED** [review]
Hi @VuDej . Great work on this class. It works perfectly as expected from the project brief. We can go ahead and merge it.

**Review by brytebee — APPROVED** [review]
Hi @VuDej 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉

**Pull request merged**